src/xdocs/checks/naming/packagename.xml

```diff
@@ -75,23 +72,17 @@ package com._checkstyle.checks_; // OK
   &lt;module name=&quot;TreeWalker&quot;&gt;
     &lt;module name=&quot;PackageName&quot;&gt;
       &lt;property name=&quot;format&quot;
-        value=&quot;^[a-z]+(\.[a-z][a-z0-9]*)*$&quot;/&gt;
+        value=&quot;^[A-Z]+(\.[a-z][a-z0-9]*)*$&quot;/&gt;
     &lt;/module&gt;
   &lt;/module&gt;
 &lt;/module&gt;
         </source>
         <p id="Example2-code">Code Example:</p>
         <source>
-package com; // OK
-package COM; // violation, name 'COM' must match pattern '^[a-z]+(\.[a-z][a-z0-9]*)*$'
-package com.checkstyle.checks; // OK
-package com.A.checkstyle.checks; // violation, name 'com.A.checkstyle' must match
-                                 // pattern '^[a-z]+(\.[a-z][a-z0-9]*)*$'
-package com.checkstyle1.checks; // OK
-package com.checkSTYLE.checks; // violation, name 'com.checkSTYLE.checks' must
-                               // match pattern '^[a-z]+(\.[a-z][a-z0-9]*)*$'
-package com._checkstyle.checks_; // violation, name 'com._checkstyle.checks_' must match
-                                 // pattern '^[a-z]+(\.[a-z][a-z0-9]*)*$'
+package com.puppycrawl.tools.checkstyle.checks.naming.packagename; // violation
+
```

Comment:
```
package com.puppycrawl.tools.checkstyle.checks.naming.packagename; 
// violation above 'Must match pattern'
```

Reply:
please do:
`// violation above 'must match pattern'`
https://github.com/checkstyle/checkstyle/blob/8e5396f11e39711d16420cbb704c5b2da232621c/src/main/resources/com/puppycrawl/tools/checkstyle/checks/naming/messages.properties#L4

Reply:
this is my question